ocular/src/error.rs

```diff
@@ -103,3 +105,15 @@ pub enum TxError {
     #[error("error broadcasting message: {0}")]
     BroadcastError(String),
 }
+
+#[derive(Debug, Error)]
+pub enum AutomatedTxHandlerError {
+    #[error("error reading file: {0}")]
+    FileIOError(String),
+    #[error("error reading toml: {0}")]
+    TomlError(String),
+    #[error("error handling key: {0}")]
+    KeyHandlingError(String),
+    #[error("error sending tx: {0}")]
+    TxError(String),
+}
```

Comment:
Nit: prefer to keep `Error` suffix out of variants

Reply:
fair enough